sites/public/pages/account/edit.tsx

```diff
@@ -0,0 +1,349 @@
+import React, { useContext, useState } from "react"
+import moment from "moment"
+import { useForm } from "react-hook-form"
+import {
+  Button,
+  Field,
+  FormCard,
+  Icon,
+  UserContext,
+  Form,
+  emailRegex,
+  t,
+  AlertBox,
+  SiteAlert,
+  ApiClientContext,
+  RequireLogin,
+  AlertTypes,
+  passwordRegex,
+} from "@bloom-housing/ui-components"
+import Link from "next/link"
+import FormsLayout from "../../layouts/forms"
+
+type AlertMessage = {
+  type: AlertTypes
+  message: string
+}
+
+const Edit = () => {
```

Comment:
This file is pretty lengthy with several different forms. Any way we could split this up into subcomponent files?

Reply:
Just checking if you saw this, thanks.

Reply:
Sorry I thought I replied! IMO I'm worried that might actually decrease readability to need to shuffle between four different files for this one view, but I definitely see where you're coming from and don't feel particularly strongly about this and could go either way :) Happy to discuss further